### ISSUE TITLE
[BUG] Cannot remove "Create single record" Airtable fields

### ISSUE THREAD

**Issue by veli-gasparovic** [PipedreamHQ/pipedream#3198]
The new Airtable integration automatically pulls all the fields from the selected table, but I cannot find a way to remove/disable some. Some Airtable fields can be automatically generated (e.g. Created At) so calling the API even with those set to EMPTY generates an error.

**Comment by DilanAthukorala** [comment]
Hi @veli-gasparovic, I tried to replicate the scenario to create a single record without entering values to some fields - (including automatically generated fields like 'created time') but I was able to create records by setting those fields to EMPTY. Could you please provide more details about the error you observed and steps to reproduce?

**Comment by veli-gasparovic** [comment]
It seems it's a problem specific to a table I have. I tried creating a new table, and the same setup works. However, when trying to add a "create single record" step associated with the problematic table, i'm getting this error now:
![image](https://user-images.githubusercontent.com/75835024/174974724-2ded4358-2306-4a27-84ff-1438f94df070.png)

I managed to work around, so just letting you know if you want to dig into.
